routes/merged/project/location/load_balancer.rb

```diff
@@ -0,0 +1,158 @@
+# frozen_string_literal: true
+
+class Clover
+  branch = lambda do |r|
+    r.get api? do
+      load_balancer_list
+    end
+
+    r.on NAME_OR_UBID do |lb_name, lb_id|
+      filter = if lb_name
+        {Sequel[:load_balancer][:name] => lb_name}
+      else
+        {Sequel[:load_balancer][:id] => UBID.to_uuid(lb_id)}
+      end
+
+      filter[:private_subnet_id] = @project.private_subnets_dataset.where(location: @location).select(Sequel[:private_subnet][:id])
+      lb = LoadBalancer.first(filter)
+
+      unless lb
+        response.status = request.delete? ? 204 : 404
+        request.halt
+      end
+
+      r.post "attach-vm" do
+        Authorization.authorize(current_account.id, "LoadBalancer:edit", lb.id)
+        required_parameters = %w[vm_id]
+        request_body_params = Validation.validate_request_body(json_params, required_parameters)
+        vm = Vm.from_ubid(request_body_params["vm_id"])
+
+        unless vm
+          response.status = 404
+          if api?
+            r.halt
+          else
+            flash["error"] = "VM not found"
+            r.redirect "#{@project.path}#{lb.path}"
+          end
+        end
+
+        Authorization.authorize(current_account.id, "Vm:view", vm.id)
+
+        if vm.load_balancer
+          flash["error"] = "VM is already attached to a load balancer"
```

Comment:
This `if vm.load_balancer` section seems web specific, when the rest of the route handles both api and web.  I'm guessing if any api request hits this, it does something we do not want.  This was a bug in the previous code, not in this PR.  So it isn't a blocker, but we should add api handling to this section.

Reply:
I removed the branch at 0b25439d6f0d2384902950edb8e320c913cfe9d5